src/activity_item_handling.cpp

```diff
@@ -379,6 +381,15 @@ void drop_on_map( Character &you, item_drop_reason reason, const std::list<item>
                 );
                 break;
         }
+
+        if( get_option<bool>( "AUTO_NOTES_DROPPED_FAVORITES" ) && it.is_favorite ) {
```

Comment:
Is there a reason it was not added outside of `if-else` block?

Reply:
I needed `it` from this if-else block.

Reply:
I see. Maybe you could gather list of all favorite items in the code block below and create overmap note for all dropped favorites after that?

```
    for( const item &it : items ) {
        here.add_item_or_charges( where, it );
        item( it ).handle_pickup_ownership( you );
    }
```

Reply:
I'll try that, when I get to my PC (tomorrow).